### ISSUE TITLE
Implement view for individual session in the "My Sessions" Dashboard

### ISSUE THREAD

**Issue by niranjan94** [fossasia/open-event-orga-server#705]
From #622.

**URL:** `/events/mysessions/ID` - shows the specific session/sessions that the user has submitted to a specific event.

Prerequisite: #703

**Comment by mariobehling** [comment]
Any screenshot?

**Comment by niranjan94** [comment]
@mariobehling yes. I'll post in a few hours. I'm out right now. Sorry.

**Comment by niranjan94** [comment]
@mariobehling this is how the UI will look like

![screencapture-localhost-5000-events-mysessions-1-1465749584671](https://cloud.githubusercontent.com/assets/2404372/15992451/72add7a0-30ea-11e6-87a7-5ee2d755543d.png)

**Comment by mariobehling** [comment]
@niranjan94 Good. Maybe move Microlocation, track etc. to the bottom.
How does the "editable version" looks like?

**Comment by niranjan94** [comment]
@mariobehling I'll move those to the bottom. 
Oh, so the user is allowed to edit his/her session ? Okay ... I didn't implement that in this yet.

**Comment by mariobehling** [comment]
So, just to confirm - this is the option for the "speaker" to view and edit his submissions.

**Comment by niranjan94** [comment]
@mariobehling yes. this is for the **speaker**. I thought they aren't allowed to edit. So, I didn't implement that in this yet But I'll add it as a separate issue.

**Comment by niranjan94** [comment]
@mariobehling which all fields is the speaker allowed to edit ?

**Comment by mariobehling** [comment]
A speaker has to submit this through a form as an answer to the Call for Speakers. @SaptakS is working on the form in step 5 of the create event wizard. The speaker should not be able to edit the location for example.

**Comment by niranjan94** [comment]
@mariobehling okay ... so, I'll push this current view now (without edit option) ... I'll add the edit option once Saptak is done with the form.

**Comment by mariobehling** [comment]
Let's keep it simple. We probably have to add more fields later. Fields are defined in step 5. How about you take out location, track etc. out and implement it later.

**Comment by niranjan94** [comment]
Okay ... Cool I'll make it such that the following fields are editable for now. Rest we can add later if required.

1. title
1. subtitle
1. abstract
1. description
